src/test/java/org/jenkinsci/plugins/githubautostatus/notifiers/StatsdNotifierTest.java

```diff
@@ -137,25 +138,21 @@ public void testIsDisabled() throws Exception {
     @Test
     public void testGetBranchPath() throws Exception {
         when(config.getStatsdHost()).thenReturn("");
-        when(config.getBranchName()).thenReturn("Shane's Branch");
-        when(config.getRepoOwner()).thenReturn("Main Folder/Sub Folder");
-        when(config.getRepoName()).thenReturn("Job Name!");
+        when(config.getExternalizedID()).thenReturn("Main Folder/Sub Folder");
```

Comment:
Should this include job name and branch also? I know it's a mock, but it would be good if our mocks matched real cases.

Reply:
Updated